File: Java/Human.java
public class Human {

    // atribut privat
    private int nik;
    private String name;
    private String gender;

    // constructor
    public Human(int nik, String name, String gender) {
        this.nik = nik;
        this.name = name;
        this.gender = gender;
    }

    public Human() {
    }

    // getter setter
    public int getNik() {
        return this.nik;
    }

    public void setNik(int value) {
        this.nik = value;
    }

    public String getName() {
        return this.name;
    }

    public void setName(String value) {
        this.name = value;
    }

    public String getGender() {
        return this.gender;
    }

    public void setGender(String value) {
        this.gender = value;
    }
}

File: PHP/Human.php
<?php

class Human
{
    // atribut private
    private $nik, $name, $gender;

    // constructor
    public function __construct()
    {
        $this->nik = "";
        $this->name = "";
        $this->gender = "";
    }

    /**
     * Get the value of nik
     */
    public function getNik()
    {
        return $this->nik;
    }

    /**
     * Set the value of nik
     *
     */
    public function setNik($nik)
    {
        $this->nik = $nik;
    }

    /**
     * Get the value of name
     */
    public function getName()
    {
        return $this->name;
    }

    /**
     * Set the value of name
     *
     */
    public function setName($name)
    {
        $this->name = $name;
    }

    /**
     * Get the value of gender
     */
    public function getGender()
    {
        return $this->gender;
    }

    /**
     * Set the value of gender
     *
     */
    public function setGender($gender)
    {
        $this->gender = $gender;
    }

    //  print
    function printHuman()
    {
        echo "Data Human :" . $this->getNik() . " --- " . $this->getName() . " --- " . $this->getGender() . "<br/>";
    }

    // destructor
    public function __destruct()
    {
    }
}
